frontend/components.js

```diff
@@ -474,6 +474,30 @@ const ConfirmTransactionDialog = connect(
   )
 })
 
+const DemoWalletWarningDialog = connect({}, actions)(({closeDemoWalletWarningDialog}) => {
+  return h(
+    'div',
+    {class: 'overlay'},
+    h(
+      'div',
+      {class: 'box center fade-in-up'},
+      h('h4', undefined, 'Public demo wallet'),
+      h(
+        'ul',
+        {class: 'public-wallet-warning-list'},
+        h('li', undefined, 'You are in Public demo wallet'),
```

Comment:
add: operating with real funds and adresses

Reply:
Added as separate liste item